packages/core/src/awsService/appBuilder/explorer/nodes/deployedNode.ts

```diff
@@ -107,7 +107,7 @@ export async function generateDeployedNode(
                         createPlaceholderItem(
                             localize(
                                 'AWS.appBuilder.explorerNode.unavailableDeployedResource',
-                                '[Failed to retrive deployed resource.]'
+                                '[Failed to retrive deployed resource. Please ensure your AWS account is connected.]'
```

Comment:
Avoid "please" in messages. It takes up valuable space, for no benefit.

```suggestion
                                '[Failed to retrive deployed resource. Ensure your AWS account is connected.]'
```